app/views/users/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<table>
+  <tr>
+    <th>
+      Username:
+    </th>
+    <th>
+      Password:
+    </th>
+  </tr>
+<% @users.each do |user| %>
+  <tr>
+    <td>
+      <%= render plain: user.username %>
```

Comment:
You only need render if you are interpreting the contents of a template.

`<%= user.username %>` should be sufficient here